### PR TITLE
mysql api

### PR DESCRIPTION
**Pull request by ally0908**

### CODE REVIEW

File: meta/material.go [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+		var aliastmp string
+		err = rows.Scan(&aliastmp)
+		if err != nil {
+			return nil, err
+		}
+		if strings.Compare(materialName, aliastmp) != 0 {
+			if allalias == "" {
+				allalias = aliastmp
+			} else {
+				allalias += ";" + aliastmp
```
Comment (polaris007) [inline meta/material.go:290]: Could we use some const variables instead of hardcoding the magic strings?

File: meta/material.go [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/go-sql-driver/mysql"
+	"strings"
+)
+
+func main() {
+	db, err := sql.Open("mysql", "root:akso@tcp(127.0.0.1:13306)/akso")
```
Comment (rubymachunyan) [inline meta/material.go]: The tcp connection and user is hard coded, could we use var?

File: meta/material.go (hunk 1 above)
Comment (ally0908) [reply meta/material.go:290]: I just added dataSourceName in a JSON file.

File: meta/material.go (hunk 2 above)
Comment (ally0908) [reply meta/material.go]: I just changed db info to one json file, called mysqlStoreConfig.json

**Review by rubymachunyan — APPROVED** [review]
The codes look good to me. Thanks a lot!

**Pull request merged**